crates/sui-framework/packages/dwallet-system/sources/dwallet.move

```diff
@@ -302,6 +326,32 @@ module dwallet_system::dwallet {
             signatures,
             sender: session.sender,
         };
-        transfer::transfer(sign_output, session.sender);
+        // TODO: Change output to be immutable instead of owned
+        transfer::transfer(sign_output, sender(session));
+    }
+
+    public(friend) fun create_malicious_aggregator_sign_output<S: store>(
+        aggregator_public_key: vector<u8>,
+        session: &SignSession<S>,
+        signatures: vector<vector<u8>>,
+        ctx: &mut TxContext
+    ) {
+        event::emit(MaliciousAggregatorEvent {
+            aggregator_public_key,
+            epoch: tx_context::epoch(ctx),
+            signatures,
+            messages: session.messages,
+            dwallet_id: session.dwallet_id,
+        });
+        let failed_sign_output = MaliciousAggregatorSignOutput {
+            id: object::new(ctx),
+            aggregator_public_key,
+            epoch: tx_context::epoch(ctx),
+            signatures,
+            messages: session.messages,
+            dwallet_id: session.dwallet_id,
+        };
+        // TODO: Change output to be immutable instead of owned
+        transfer::transfer(failed_sign_output, sender(session));
```

Comment:
I think it should be useful to include the session ID in which the malicious behavior occurred